Makefile

```diff
@@ -542,8 +543,13 @@ e2e.build.centos-7: centos7 docker.clean-latest-ami
 e2e.build.centos-7-offline:
 	$(MAKE) os-packages-artifacts pip-packages-artifacts
 	$(MAKE) devkit.run WHAT="make save-images"
-	$(MAKE) devkit.run WHAT="make centos7 ADDITIONAL_OVERRIDES=overrides/offline.yaml"
+	$(MAKE) devkit.run WHAT="make infra.aws.init"
+	$(MAKE) devkit.run WHAT="make infra.aws.create"
+	$(MAKE) devkit.run WHAT="make packer-custom-vpc-override.yaml"
+	$(MAKE) devkit.run WHAT="make build"
+	$(MAKE) devkit.run WHAT="./bin/konvoy-image build images/ami/centos-7.yaml -v ${VERBOSITY} --overrides=overrides/offline.yaml --overrides=packer-custom-vpc-override.yaml"
```

Comment:
I faced same issue of difficulty in providing multiple override files dynamically to makefile targets. This is one way to address it https://github.com/mesosphere/konvoy-image-builder/pull/220 . PTAL